docs-source/content/userguide/managing-domains/domain-in-image/base-images/_index.md

```diff
@@ -88,38 +88,51 @@ Oracle Container Registry.
 
 #### Creating a custom image with patches applied
 
-The Oracle WebLogic Server Kubernetes Operator requires patch 29135930.
-This patch does have some prerequisite patches that will also need to be applied. The standard image `container-registry.oracle.com/middleware/weblogic:12.2.1.3` already has all of these patches applied.
+The Oracle WebLogic Server Kubernetes Operator and WebLogic Server 12.2.1.3 image require patch 29135930.
```

Comment:
Elsewhere we \use 5 digit version, but here we use 4 - should be consistent.  
require -> requires